src/Calculator/Calculator.vcxproj.filters

```diff
@@ -471,15 +463,13 @@
     <Page Include="Views\DateCalculator.xaml">
       <Filter>Views</Filter>
     </Page>
-    <Page Include="Views\TitleBar.xaml">
-      <Filter>Views</Filter>
-    </Page>
     <Page Include="Views\MemoryListItem.xaml">
       <Filter>Views</Filter>
     </Page>
     <Page Include="Views\OperatorsPanel.xaml">
       <Filter>Views</Filter>
     </Page>
+    <Page Include="Views\TitleBar.xaml" />
```

Comment:
This should go under the `Views` filter with the rest of the XAML pages.

Reply:
@rudyhuyn, thoughts?

Reply:
You are right! (if only VS were able to handle file creation and filters a little better)

Reply:
I agree...